src/index.ts

```diff
@@ -46,3 +56,14 @@ export const Ludex = {
   OrganizationScoped,
   ClientScoped,
 };
+
+export {
+  Chain,
+  PayoutType,
+  PayoutState,
+  Environment,
+  ChallengeState,
+  RedeemType,
```

Comment:
Export zod enum so that typescript users can use corresponding zod enum when using our SDK. Otherwise, typescript will throw error during compilation

<img width="1536" alt="image" src="https://github.com/Ludex-Labs/ludex-sdk-js/assets/105812701/589670c8-6468-4dde-b641-d1d7e37da7c0">

Reply:
Example usage: 

<img width="727" alt="image" src="https://github.com/Ludex-Labs/ludex-sdk-js/assets/105812701/f2decd53-6255-4d20-8ae2-2cf65677e9fe">

Reply:
if we export the enums it might make more sense for us to use the native enums just to make it more ergonomic